PR: https://github.com/odoo/odoo/pull/

From: ca06bfaf4f7888e3e9c46ef896ede0b35cbb1658
From: svs-odoo
Date: 2019-10-25 13:20:38

Structural Changes: 10
Total Changes: 23

[IMP] product_expiry: visual changes

Changes the label and tooltip of expiration date related fields.

Task #1938656

================================= pseudo patch: =================================

--- a/addons/product_expiry/models/product_product.py
+++ b/addons/product_expiry/models/product_product.py
@@ -10,11 +10,16 @@ class ProductTemplate(models.Model):
     use_expiration_date = fields.Boolean(string='Expiration Date',
         help='When this box is ticked, you have the possibility to specify dates to manage'
         ' product expiration, on the product and on the corresponding lot/serial numbers')
-    expiration_time = fields.Integer(string='Product Life Time',
-        help='Number of days before the goods may become dangerous and must not be consumed. It will be computed on the lot/serial number.')
-    use_time = fields.Integer(string='Product Use Time',
-        help='Number of days before the goods starts deteriorating, without being dangerous yet. It will be computed using the lot/serial number.')
-    removal_time = fields.Integer(string='Product Removal Time',
-        help='Number of days before the goods should be removed from the stock. It will be computed on the lot/serial number.')
-    alert_time = fields.Integer(string='Product Alert Time',
-        help='Number of days before an alert should be raised on the lot/serial number.')
+    expiration_time = fields.Integer(string='Expiration Time',
+        help='Number of days after the receipt of the products (from the vendor'
+        ' or in stock after production) after which the goods may become dangerous'
+        ' and must not be consumed. It will be computed on the lot/serial number.')
+    use_time = fields.Integer(string='Best Before Time',
+        help='Number of days before the Expiration Date after which the goods starts'
+        ' deteriorating, without being dangerous yet. It will be computed on the lot/serial number.')
+    removal_time = fields.Integer(string='Removal Time',
+        help='Number of days before the Expiration Date after which the goods'
+        ' should be removed from the stock. It will be computed on the lot/serial number.')
+    alert_time = fields.Integer(string='Alert Time',
+        help='Number of days before the Expiration Date after which an alert should be'
+        ' raised on the lot/serial number. It will be computed on the lot/serial number.')

--- a/addons/product_expiry/models/production_lot.py
+++ b/addons/product_expiry/models/production_lot.py
@@ -9,7 +9,7 @@ class StockProductionLot(models.Model):
 
     use_expiration_date = fields.Boolean(
         string='Use Expiration Date', related='product_id.use_expiration_date')
-    expiration_date = fields.Datetime(string='End of Life Date',
+    expiration_date = fields.Datetime(string='Expiration Date',
         help='This is the date on which the goods with this Serial Number may become dangerous and must not be consumed.')
     use_date = fields.Datetime(string='Best before Date',
         help='This is the date on which the goods with this Serial Number start deteriorating, without being dangerous yet.')
